src/oc/web/components/cmail.cljs

```diff
@@ -365,7 +365,14 @@
                 {:class (when long-tooltip "long-tooltip")}
                 [:button.mlb-reset.close-bt
                   {:on-click #(do
-                                (autosave s)
+                                (if (and (nil? (:video-id cmail-data))
+                                         (empty? (:attachments cmail-data))
+                                         (clojure.string/blank?
+                                          (:headline cmail-data))
+                                         (clojure.string/blank?
+                                          (:body cmail-data)))
```

Comment:
I see we are using this check a couple of times, would be good to have a function in a utility namespace to check if a post has something good to be saved for:
- has attachments or
- has a video or
- has a headline or
- has a body

WYT?

We could reuse that also in `oc.web.actions.activity/remove-video`